api/types/stats.go

```diff
@@ -162,7 +162,7 @@ type Stats struct {
 
 	// Shared stats
 	CPUStats    CPUStats    `json:"cpu_stats,omitempty"`
-	PreCPUStats CPUStats    `json:"precpu_stats,omitempty"` // "Pre"="Previous"
+	PreCPUStats *CPUStats   `json:"precpu_stats,omitempty"` // "Pre"="Previous"
```

Comment:
This is because I can't find a reasonable method to detect if the `PreCPUStats` is valid or not, then I change it to a pointer, it will be valid if it's not nil, vice versa.